src/Components/ServiceScene/Breakdowns/PatternsViewTableScene.tsx

```diff
@@ -204,62 +206,76 @@ const getTablePatternTextStyles = () => {
     overflowWrap: 'break-word',
   });
 };
-const vizStyles = {
-  tablePatternTextDefault: css({
-    fontFamily: theme.typography.fontFamilyMonospace,
-    minWidth: '200px',
-    maxWidth: '100%',
-    overflow: 'hidden',
-    overflowWrap: 'break-word',
-    fontSize: theme.typography.bodySmall.fontSize,
-    wordBreak: 'break-word',
-  }),
-  countTextWrap: css({
-    textAlign: 'right',
-    fontSize: theme.typography.bodySmall.fontSize,
-  }),
-  tableTimeSeriesWrap: css({
-    width: '230px',
-    pointerEvents: 'none',
-  }),
-  tableWrap: css({
-    // Override interactive table style
-    '> div': {
-      // Need to define explicit height for overflowX
-      height: 'calc(100vh - 450px)',
-      minHeight: '470px',
-    },
-    // Make table headers sticky
-    th: {
-      top: 0,
-      position: 'sticky',
-      backgroundColor: theme.colors.background.canvas,
-      zIndex: theme.zIndex.navbarFixed,
-    },
-  }),
-  tableTimeSeries: css({
-    height: '30px',
-    overflow: 'hidden',
-  }),
+
+const getTableStyles = (theme: GrafanaTheme2) => {
```

Comment:
This fixes a bug @matryer found if you switch the theme without refreshing the page the background colors wouldn't update